lib/redmine_email_inline_images/mail_handler_patch.rb

```diff
@@ -15,11 +15,11 @@ module InstanceMethods
       # an issue created by an email request
       def plain_text_body_with_email_inline_images
         return @plain_text_body unless @plain_text_body.nil?
-        part = email.text_part || email.html_part || email
+        part = email.html_part || email.text_part || email
```

Comment:
The plain text part of the email never contains inline images, so the plugin should prefer using the HTML part.